MANIFEST.in

```diff
@@ -5,7 +5,6 @@ include AUTHORS.rst
 include HISTORY.rst
```

Comment:
remove this line since we deleted HISTORY.rst from the repo

Reply:
changed to CHANGELOG.md